src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import { Navigate, HashRouter as Router } from 'react-router-dom';
+import { Routes, Route } from 'react-router-dom';
+import { App } from './App';
+import HomePage from './pages/HomePage';
+import PeoplePage from './pages/PeoplePage';
+
+const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="people/:slug?">
+            <Route index element={<PeoplePage />} />
+            <Route path=":slug" element={<PeoplePage />} />
+          </Route>
+          <Route path="*" element={<h1 className="title">Page not found</h1>} />
+          <Route path="home" element={<Navigate to="/" replace={true} />} />
```

Comment:
```suggestion
          <Route path="home" element={<Navigate to="/" replace />} />
```

just one small update